docsbox/Dockerfile

```diff
@@ -1,29 +1,38 @@
 FROM ubuntu:20.04
 ENV TZ Europe/Helsinki
 
-# install prequisited
-RUN apt-get update && apt-get install -y software-properties-common \
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime \
+    && echo $TZ > /etc/timezone
+
+# Install prequisited
+RUN apt-get update && apt-get install -y \
+    software-properties-common debconf wget curl \
+    locales build-essential dialog apt-utils \
     && rm -rf /var/lib/apt/lists/* \
     && apt-get clean
 
+# Install Libreoffice
+RUN add-apt-repository ppa:libreoffice/ppa
 # Define fonts
 RUN echo debconf debconf/frontend select Noninteractive | debconf-set-selections \
     && echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN add-apt-repository ppa:libreoffice/ppa
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime \
-    && echo $TZ > /etc/timezone
+RUN apt-get update \
+    && apt-get install -y fontforge cabextract fonts-dejavu-extra \
+    && wget https://gist.github.com/maxwelleite/10774746/raw/ttf-vista-fonts-installer.sh -q -O - | bash \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
 
 # Fetch dependencies
 RUN apt-get update \
```

Comment:
No need to re-run update here if it's run above since there are no edits in the repositories